lib/job.js

```diff
@@ -14,7 +14,13 @@ var Job = module.exports = function Job(args) {
   delete args.agenda;
 
   // Process args
-  args.priority = parsePriority(args.priority) || 0;
+
+  // allow priority overrides (e.g. `job.now('foo', { priority: 'low' })`)
```

Comment:
@lushc Ah, be careful - you needed to wrap `schedule(now)` and `schedule(now + 200)` with `schedule(new Date(now))` and `schedule(new Date(now + 200))`.  Agenda by default doesn't parse a timestamp as a date. 

This seems to be an actual bug - shouldn't we support passing a timestamp from `Date.now()` or `new Date().getTime()`? cc @OmgImAlexis